plugin/src/withMapLibre.ts

```diff
@@ -125,9 +125,28 @@ export function setExcludedArchitectures(project: XcodeProject): XcodeProject {
       typeof buildSettings?.PRODUCT_NAME !== 'undefined'
     ) {
       buildSettings['"EXCLUDED_ARCHS[sdk=iphonesimulator*]"'] = '"arm64"';
+
+      /**
+       * Delete Duplicate Signature File
+       * https://github.com/CocoaPods/CocoaPods/issues/12022
+       */
+      const shellScript = `if [ "$XCODE_VERSION_MAJOR" = "1500" ]; then
+      echo "Remove signature files (Xcode 15 workaround)";
+      rm -rf "$BUILD_DIR/Release-iphoneos/Mapbox.xcframework-ios.signature";
```

Comment:
If this is still needed after SDK 10.2.0, I've tested the fix and it seems like it only works for Release-iphoneos config builds.  We could use `CONFIGURATION_BUILD_DIR` instead of hard coding the config to cover debug builds and build targeting other platforms:

```sh
rm -rf "$CONFIGURATION_BUILD_DIR/Mapbox.xcframework-ios.signature";
```

Reply:
As far as I can tell, it is. The SDK 10.2.0 change was Android only.  The IOS 6.0 (metal) release isn't included and has a lot of breaking changes that will need to be addressed separately. I will make this change and rebase off main and then we can merge.

Reply:
Hi, is there a way I can test these changes without manually building? I tried to install from your fork but that doesn't work because of conflicts with yarn and husky (I don't use them).

Reply:
Hi @jxeeno @tyrauber. Thanks for your quick replies. I am indeed using Expo with EAS but my builds keep failing for SDK 50. I also tried updating the plugin with `patch-package` but on EAS it still crashes on the pod install. Guess I'll have to do it manually, or is it possible to release a new alpha with these changes?

Reply:
@aLemonFox,  Try that last approach.  

Create a file `plugins/withRemoveSignatureFiles.js` with that code, and add the plugin to app.json.
```
{
    "plugins": [
      ...
      "@maplibre/maplibre-react-native",
      "./plugins/withRemoveSignatureFiles.js",
    ]
 }
    ```
    
I have successfully built on EAS with this approach for both Android and iOS. If you still have a problem post the EAS error.

I'd love to see this get merged and tagged as 10.0.0.alpha.2, but I don't have access. @sarahsporck, @ianthetechie, can we merge this?

Reply:
I added the plugin but on EAS it still fails with
```
xcodebuild: error: “Mapbox.xcframework-ios.signature” couldn’t be copied to “Signatures” because an item with the same name already exists.: The operation couldn’t be completed. File exists
2024-02-08 02:51:27.009 xcodebuild[5489:15802] Writing error result bundle to /var/folders/p2/2788nfhd7xq3pnnk81v00sf00000gn/T/ResultBundle_2024-08-02_02-51-0027.xcresult
```

My setup looks like this with the plugins specified as you posted.
```
root
├── plugins
│   └── withRemoveSignatureFiles.js
└── app.json
```
```json
// app.json
"plugins": [
      "expo-localization",
      "@maplibre/maplibre-react-native",
      "./plugins/withRemoveSignatureFiles.js",
      "expo-router"
    ],
```

Reply:
That's really odd. It seems like the plugin isn't executing. And you ran `expo prebuild` before pushing to EAS?  Are you on version 10.0.0.alpha.1? 

Maybe try this variant in the plugin:
```
rm -rf "$CONFIGURATION_BUILD_DIR/Mapbox.xcframework-ios.signature";
```

Reply:
I cannot use `expo prebuild` for iOS as I am on windows unfortunately.